api/nodes/serializers.py

```diff
@@ -63,10 +40,12 @@ class NodeSerializer(JSONAPISerializer):
                                                             'public and private nodes. Administrators on a parent '
                                                             'node have implicit read permissions for all child nodes',
                               )
-    # TODO: finish me
 
-    class Meta:
-        type_ = 'nodes'
+    registration = ser.BooleanField(source='is_registration', read_only=True)
```

Comment:
True, but having a schema doesn't obviate the need for informative field names. Having `is_` makes the type clear without having to refer to the schema.